src/main/java/org/hmhb/user/HmhbUser.java

```diff
@@ -0,0 +1,65 @@
+package org.hmhb.user;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+import org.apache.commons.lang3.builder.ToStringBuilder;
+
+@Entity
+public class HmhbUser {
```

Comment:
It would be nice if we also stored the user's profile picture for use in the UI.

I believe it is provided in the basic user profile from google.